draft-decraeneginsberg-lsr-isis-fast-flooding.xml

```diff
@@ -363,32 +363,31 @@
 
             <t>LSPs</t>
 
-            <t>Complete Sequence Number PDUs (CSNPs) and Partial Sequence
-            Number PDUs (PSNPs)</t>
+            <t>Complete Sequence Number PDUs (CSNPs) and PSNPs</t>
           </list>Implementations today may prioritize the reception of Hellos
         over LSPs and SNPs in order to prevent a burst of LSP updates from
         triggering an adjacency timeout which in turn would require additional
         LSPs to be updated.</t>
 
-        <t>SNPs serve to acknowledge or trigger the transmission of specified
+        <t>CSNPs and PSNPs serve to acknowledge or trigger the transmission of specified
```

Comment:
s/acknowledge or trigger/trigger or acknowledge
This matches the order in which CSNP/PSNP are mentioned.

Reply:
Accepted.